app/src/main/java/com/example/footprnt/Map/PostDetailActivity.java

```diff
@@ -12,6 +6,7 @@
 import android.widget.ImageView;
 
 import com.example.footprnt.Map.Util.MapUtil;
+import com.example.footprnt.Map.Util.PostAdapter;
```

Comment:
It looks like this import is the only change in this file. Why add the import if PostAdapter isn't added to this file?